boot/bootutil/include/bootutil/sha256.h

```diff
@@ -61,20 +61,20 @@ typedef mbedtls_sha256_context bootutil_sha256_context;
 static inline void bootutil_sha256_init(bootutil_sha256_context *ctx)
 {
     mbedtls_sha256_init(ctx);
-    mbedtls_sha256_starts(ctx, 0);
+    (void)mbedtls_sha256_starts_ret(ctx, 0);
```

Comment:
I think this is fine for this commit, but I'm guessing these are made to return errors for some reason. Perhaps if there is a failure with a hardware crypto module. I wonder if we should consider handling the errors (not that there is much we can do).

Reply:
I did not want to apply changes up the stack; there are not really many errors that could happen for us; also tinycrypt does not return errors.